index.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     'no-restricted-syntax': 'off',
     'no-return-await': 'warn',
     'no-template-curly-in-string': 'warn',
-    'no-undef': 'warn',
+    'no-undef': 'error',
```

Comment:
will need exceptions for document, window etc. but I'm happy to make that a ui problem